### DISCUSSION TITLE
Favicon Not Updating on Vercel Hosted App

### DISCUSSION THREAD

**Discussion by sudarshanj01** [vercel/next.js#65973] [Help] [answered]
### Summary

I have added a favicon.ico to my Next.js app located at app/favicon.ico. The favicon updates correctly when running the app locally, but it does not update on the Vercel hosted version of the app.

Steps taken:

Added favicon.ico to app/favicon.ico.
Confirmed the update locally.

Deployed the app to Vercel.
Expected Behavior:
The favicon should be updated on the Vercel hosted app.

Actual Behavior:
The favicon remains unchanged on the Vercel hosted app.

Any guidance on resolving this issue would be appreciated.

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Can you reach for the favicon URL path normally?

If so, then it might be a problem with your browser. Try another browser, or incognito mode. If that works, then clear your browser cache. https://fabricdigital.co.nz/blog/how-to-hard-refresh-your-browser-and-clear-cache

**Reply by sudarshanj01** [reply]
It worked! Thank you

**Reply by Strong-Ari** [reply]
I have tried many browsers but nothing works

**Comment by hasan-mohammad-bd** [comment]
Hi, 
Take a look https://github.com/vercel/next.js/discussions/65938#discussioncomment-9492301, Already solved the issue here
